### DISCUSSION TITLE
What do non-special bracketed sequences mean in the dictionary?

### DISCUSSION THREAD

**Discussion by mattfbacon** [opensteno/plover#1605] [Q&A and Support] [answered]
I found eg `"OBGS/A*PB": "{oxan}",` as an entry in the dictionary. What does this mean/do?

**Comment by stenoknight** [comment, marked as answer]
I think this is an artifact of the time when I used to use Eclipse.
Sometimes it would make weird erroneous dictionary entries that looked like
this. Feel free to delete or overwrite.

**Reply by mattfbacon** [reply]
Thanks. Do they behave the same as the unbracketed equivalent?
